### ISSUE TITLE
Find a smaller replacement to puppeteer for tests

### ISSUE THREAD

**Issue by XhmikosR** [svg-sprite/svg-sprite#597]
We previously used svg2png but it's not maintained. Puppeteer on the other hand is huge.

**Comment by Kreeg** [comment]
Puppeteer also has this bug https://github.com/puppeteer/puppeteer/issues/8009, leaving me without a chance to unplug the power connector
There is two candidates now:
1. playwright (svgo uses it)
2. webdriverjs

Any help is welcome

**Comment by XhmikosR** [comment]
1. playwright also downloads huge binaries and it's even bigger, it just caches the binaries system-wide...
2. haven't used this

Looking at the svg2png PRs, I see there was a PR to switch to puppeteer there too. So, maybe we are stuck, in that case we'll live with puppeteer.

**Comment by yisibl** [comment]
Switching to puppeteer-core is a little better, and you can use the Chrome path already installed locally.

**Comment by XhmikosR** [comment]
If you could make a PR, that would help!

I wonder if there's a similar solution for other browsers too. Maybe playwright supports using the existent installed browsers?

Either way, this should be an improvement :)

**Comment by yisibl** [comment]
@weolar has a [miniblink](https://github.com/weolar/miniblink49) project, a streamlined browser kernel that might make a difference.

**Comment by weolar** [comment]
yes, i'm working for miniblink now. bug it's linux version has some bug to fix

**Comment by XhmikosR** [comment]
It would really help if docs were written in English too.
